monai/transforms/intensity/array.py

```diff
@@ -2161,3 +2157,99 @@ def __call__(self, img: torch.Tensor) -> torch.Tensor:
                 img = IntensityRemap(self.kernel_size, self.R.choice([-self.slope, self.slope]))(img)
 
         return img
+
+
+class ForegroundMask(Transform):
```

Comment:
We should be changing the deformation transforms to have one class for 2D and 3D but regardless if we have a transform that is definitely only 2D it should say so in the name. It'll have to be something like `Foreground2DMask` since the name can't start with a number.

Reply:
`Foreground2DMask` looks good to me.

Thanks.

Reply:
Thanks for your input. It should work for any dimensional input array. I don't know why did you get the impression that it works with only 2D. To make sure, I added a test case to include a 3D image.

Reply:
If it works for 3D, that's cool! Please add the 3D test.

Thanks.